packages/api-core/src/resources/organizations.js

```diff
@@ -45,7 +45,11 @@ export default class AvOrganizations extends AvApi {
   }
 
   async postGet(data, config, additionalPostGetArgs) {
-    if (additionalPostGetArgs) {
+    if (additionalPostGetArgs && additionalPostGetArgs.resourceIds) {
```

Comment:
`data.permissionId` should still be set if no `resourceIds` are set in `additionalPostGetArgs`. We may just be filtering by permission.